shell/platform/windows/client_wrapper/testing/stub_flutter_windows_api.h

```diff
@@ -43,11 +43,11 @@ class StubFlutterWindowsApi {
   // Called for FlutterDesktopDestroyView.
   virtual void DestroyViewController() {}
 
-  // Called for FlutterDesktopProcessMessages
-  virtual uint64_t FlutterDesktopProcessMessages() { return 0; }
+  // Called for FlutterDesktopProcessMessages.
+  virtual uint64_t ProcessMessages() { return 0; }
```

Comment:
Isn't it better to use the same name for an easier reference to the original call?

Reply:
The naming pattern (established in the GLFW version of this, and already for other methods here) is that it's the name with the unnecessary (because it's in a class, not a flat namespace) `FlutterDesktop` prefix. I just didn't catch these in initial review of the Windows version, and am fixing them now.

Reply:
👍